packages/ButtonGroup/src/components/ButtonItem/ButtonItem.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import * as sc from "./ButtonItem.styles";
+
+const propTypes = {
+  /** Content label of the button to be displayed. */
+  children: PropTypes.node,
+
+  /** The button label of the item to be used as a key in the group. */
+  label: PropTypes.node.isRequired,
+
+  /** If the button shows or hides the icon */
+  hasIcon: PropTypes.bool,
+
+  /** If the item is active or on selected state */
+  isActive: PropTypes.bool,
+};
+
+const defaultProps = {
+  children: null,
+  hasIcon: false,
+  isActive: false,
+};
+
+const ButtonItem = props => {
+  const { children, label, isActive, hasIcon } = props;
+
+  return (
+    <sc.ButtonItem key={label} data-pka-anchor="button" {...props} kind="flat">
```

Comment:
The `label` isn't actually being used, in your example anyway. You don't really need the `key` here either, you just need it in ButtonGroup, where you do a `map()`.

Reply:
Also, you don't need the `pka-anchor` and `kind` here if they're already in ButtonGroup.  But actually, I'm not sure which place makes the _most_ sense 🤔